src/StandardGovernor.sol

```diff
@@ -228,6 +228,11 @@ contract StandardGovernor is IStandardGovernor, BatchGovernor {
 
     /* ============ View/Pure Functions ============ */
 
+    /// @inheritdoc IGovernor
+    function COUNTING_MODE() external pure returns (string memory) {
+        return "support=against,for&quorum=for&success=standard";
```

Comment:
how about `success=majority` ?

Reply:
Yes, I just need to check what happens when `yes == no`. Its the difference between `majority` and `majoirtyPlusOne`?